src/models/size.rs

```diff
@@ -0,0 +1,24 @@
+use serde::{Deserialize, Serialize};
+
+/// Helper enum to represent a size which can be
+/// an absolute pixel value or a relative percentage value
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Copy)]
+#[serde(untagged)]
+pub enum Size {
```

Comment:
In retrospect, I think it is a bit weird that any int is automatically assumed to be a pixel value, and any float is assumed to be a percentage... Maybe in the long run, it would be cleaner to allow setting a unit and parse from that, similar to CSS. Where an absent unit would default to pixel.
`250` => `Pixel(250)`
`250px` => `Pixel(250)`
`250%` => `Percentage(250)`

Reply:
Maybe keep the float with `2.5 => Fraction(2.5)`?
I Actually like the distinction of absolute sizes in int and relative sizes in float.

All in all, the `size` model is a really great achievement of this PR and I can see use of it in many other config options, like margins, gutters, scratch pad dimensions and position, default floating dimensions etc.
:partying_face: